spec/requests/api/v1/users/omniauth_callbacks_controller_spec.rb

```diff
@@ -17,9 +17,46 @@
           }
         )
 
-        post '/api/v1/users/auth/google_oauth2/callback'
+        post "#{api_v1_user_google_oauth2_omniauth_callback_path}?state=%7B%22client_id%22%3A%22#{application.uid}k%22%7D"
 
         expect(response).to have_http_status(:success)
+        expect(JSON.parse(response.body)['access_token']).not_to be_nil
```

Comment:
```suggestion
        expect(JSON.parse(response.body)['access_token']).not_to be_blank
```

Or `.to be_present`. As in case the test return `{ access_token: '' }`, you would have a false positive (i.e. code broken, test successful)